pkgs/applications/networking/protonmail-bridge/default.nix

```diff
@@ -67,6 +78,13 @@ in stdenv.mkDerivation rec {
       $out/lib/protonmail-bridge
   '';
 
+  postInstall = ''
+    wrapProgram $out/bin/spectral \
+      --set QML2_IMPORT_PATH "${qml2ImportPath}"
```

Comment:
Why is this needed when we're using `wrapQtAppsHook`? The issue was most likely caused by none of the qt deps being in `buildInputs` so the hook couldn't collect anything.

Also you should switch this expression to use `mkDerivation` which will automatically add `wrapQtAppsHook`.

Reply:
ah yeah - forgot to cleanup. my bad 👍